snafu/utils/wrapper_factory.py

```diff
@@ -11,7 +11,11 @@
 from snafu.ycsb_wrapper.ycsb_wrapper import ycsb_wrapper
 from snafu.scale_openshift_wrapper.scale_openshift_wrapper import scale_openshift_wrapper
 from snafu.stressng_wrapper.stressng_wrapper import stressng_wrapper
+<<<<<<< HEAD
```

Comment:
@smalleni FYI extra bits from the rebase made it in

Reply:
Serves me right for rebasing during a meeting.